src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -823,6 +831,11 @@ def add_electrical_series(
         ecephys_mod.data_interfaces["LFP"].add_electrical_series(es)
 
 
+add_electrical_series.__doc__ = add_electrical_series.__doc__.format(
+    compression_options_docstring=compression_options_docstring, iterator_options_docstring=iterator_options_docstring
+)
```

Comment:
```suggestion
add_electrical_series.__doc__ = add_electrical_series.__doc__.format(
    compression_options_docstring=compression_options_docstring, iterator_options_docstring=iterator_options_docstring
)
```

I'm sure this works, but it's a bit counter-intuitive to anyone who is not expecting the doc to be constructed dynamically. Would it instead be possible to do this as 

```
from .docstrings import compression_docstring, iterator_options_docstring

def my_func(my_arg: int):
    """
    my_arg: int
        desc
    """ + compression_docstring + iterator_options_docstring
    pass
```

Reply:
Alas, the actual specifcatiton of docstrings at time of function definition must be a raw string literal, and cannot perform string operations on the same line.

Simple example to play with:

```python
def test_fun():
    """
    Testing123
    """ + "abc" + "def"
    pass

>>> test_fun?  # or help(test_fun), or print(test_fun.__doc__)
```

Thus modifications must take place after function definition, as done here and sister PR #273 (though I prefer most the util function in #273)

Reply:
Note in the example above, there is no docstring because the string operations nullified it. Remove the '+' temporarily to see the `Testing123` docstring

Reply:
Isn't this a use case for a decorator?

Reply:
hmm thwarted